bcda/api/v1/api.go

```diff
@@ -113,7 +113,7 @@ func JobStatus(w http.ResponseWriter, r *http.Request) {
 	defer database.Close(db)
 
 	var job models.Job
-	err := db.Find(&job, "id = ?", jobID).Error
+	err := db.First(&job, "id = ?", jobID).Error
```

Comment:
Nit - Since you're looking by the primary key, you don't need to specify the `id = ?` argument. 

Ref: https://gorm.io/docs/query.html#Retrieving-objects-with-primary-key